tests/integration/dual-channel-replication.tcl

```diff
@@ -23,14 +23,20 @@ proc get_client_id_by_last_cmd {r cmd} {
     return $client_id
 }
 
-# Wait until the process enters a paused state, then resume the process.
-proc wait_and_resume_process idx {
+# Wait until the process enters a paused state.
+proc wait_process_pause idx {
```

Comment:
Is "paused" more clear in this function name?

```suggestion
proc wait_process_paused idx {
```

Reply:
yes, it is clearer